ui/apps/platform/src/Containers/Login/TestLoginResultsPage.js

```diff
@@ -56,6 +56,12 @@ function getMessage(response) {
                 <h2 className="font-700">User Attributes:</h2>
                 <ul className="list-none">{displayAttributes}</ul>
             </p>
+            <p className="pb-2 mb-2 border-b border-success-700">
+                <span className="font-700" id="user-id-label">
+                    IdP Token:
+                </span>{' '}
+                <span aria-labelledby="user-id-label">{response?.idpToken}</span>
```

Comment:
It looks like the `id` and corresponding `aria-labeledby` attributes were just copied from one of the blocks above. 

Aria-labelledby is used to point to which other element has an explanation for the element it is on. Since ID attributes should be unique on a page, this duplication would confuse assistive technologies like screen readers.

Just change this to some other, unique ID, like
```suggestion
                <span className="font-700" id="user-idp-token">
                    IdP Token:
                </span>{' '}
                <span aria-labelledby="user-idp-token">{response?.idpToken || 'Not available'}</span>
```

Not, I also suggested some default text to print if the `idpToken` is not there, again as an aid to screen readers.

Reply:
It'd be great to not show this to begin with in case the IdP token isn't set, especially because it will be only set within the backend if an OIDC auth provider is used.